tests/schema/tests.py

```diff
@@ -178,6 +178,34 @@ def test_fk(self):
             self.fail("No FK constraint for author_id found")
 
     @skipUnlessDBFeature('supports_foreign_keys')
+    def test_fk_to_proxy(self):
+        "Tests that creating a FK to a proxy model creates database constraints."
```

Comment:
I have been in the habit of omitting the "Tests that" prefix and simply stating the expected behavior.